docs/configuration/storage/sqlite.md

```diff
@@ -20,6 +20,7 @@ requires you setup an external database.
 
 ```yaml
 storage:
+  encryption_key: a_very_important_secret
```

Comment:
There's no reference to the encryption key option linking back to the other doc for this this storage type.